### DISCUSSION TITLE
如何判断应该在哪个版块发帖？

### DISCUSSION THREAD

**Discussion by adoyle-h** [just-talks/tech-talks#13] [2.Q&A] [answered]
初来乍到，担心发错帖子违反群规。

**Comment by adoyle-h** [comment, marked as answer]
如果你想讨论、分享、提问的是有关具体的编程语言。请查看[版块目录](https://github.com/just-talks/tech-talks#%E7%89%88%E5%9D%97%E7%9B%AE%E5%BD%95)，到对应的编程语言版块下发帖。

如果你想分享一个工具或书籍，要看其使用者是哪个群体。如果是大众程序员，就发到 [tech-talks 的版块](https://github.com/just-talks/tech-talks/discussions)。如果使用群体是某个细分领域的，比如 rust 领域的程序员才会用的到，那么就发到 [rust 版块](https://github.com/just-talks/rust/discussions)。

如果你想分享的是一个库 (Library) 或者框架，你应该发到对应编程语言的版块。因为这个工具需要用这个编程语言才能使用。

不必担心发错帖子。没有惩罚。管理员会帮你把帖子移到合适的位置，并通知你。

**Comment by kalxd** [comment]
如果我想对比两门语言特性，该发布到哪里呢？例如列举Haskell和Rust的异同点。

**Reply by adoyle-h** [reply]
我设想的是任意版块里新建帖子，另一个也发但只发链接。比如先在 Haskell 发了，再到 Rust 版块发个帖，标题跟 Haskell 的一样，内容是简要说明加上 Haskell 那篇的链接，让感兴趣的人在那个帖子下讨论。
然后再点击 Lock conversation 把讨论锁了。我不知道 Author 有没有权限锁，如果没有，就打个 `Locked` Label，管理员看到后会锁上的。保证大家只在一个帖子下讨论。
就像这样：https://github.com/just-talks/golang/discussions/2
